eknigge/GuideWalker
# Write to firestore

# Description of Change

This PR addresses the following issues:

Closes #30 - Firebase - Write to Database

Adds a "Create Marker" button on the side drawer. When the button is selected, it takes you to a form to fill in the details for a new marker. After filling in the details and selecting submit, the data will be updated in the Firestore. 

<img width="324" alt="Screen Shot 2021-12-01 at 1 48 16 PM" src="https://user-images.githubusercontent.com/47171828/144319439-188382fc-565d-45fb-a506-a42b59abf3a8.png">


<img width="339" alt="Screen Shot 2021-12-01 at 1 47 50 PM" src="https://user-images.githubusercontent.com/47171828/144319381-3cb3f56e-de8b-47d7-90f6-7e994d3f9e5e.png">
## How to test

1. Run the app.
2. Open the side drawer with the chamber button on the top left.
3. Select Create Marker.
4. Fill up the form with details for a new marker.
5. Select Submit.
6. Check Firestore for a marker.
7. Delete the newly created marker. Still unable to write to the location field properly which will cause an error if left in the Firestore. The error will be resolved in a new issue.

## Checklist

- [ ] Did you request a review from someone?
